backend/scripts/block_traffic.sh

```
@@ -0,0 +1,56 @@
+# 1. ALB가 타겟 등록 취소 (등록 취소 지연 시간 동안 기존 연결 유지)
+# 2. 이 스크립트에서 SIGTERM으로 Graceful Shutdown 수행
+# 3. ApplicationStop에서 남은 프로세스 강제 종료
+# ==========================================
+
+echo ""
+echo "☕ 1. Spring Boot 애플리케이션 Graceful Shutdown 시작..."
+
+if [ -f "app/coffee-shout.pid" ]; then
+    PID=$(cat app/coffee-shout.pid)
+
+    if ps -p $PID > /dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)